OpenRA.Mods.Common/EditorBrushes/EditorCopyPasteBrush.cs

```diff
@@ -233,8 +233,7 @@ public void Do()
 
 			if (copyFilters.HasFlag(MapCopyFilters.Actors))
 			{
-				var removeActors = dest.SelectMany(editorLayer.PreviewsAt).Distinct().ToList();
-				foreach (var preview in removeActors)
+				foreach (var preview in dest.SelectMany(editorLayer.PreviewsAt).ToHashSet())
```

Comment:
Just `Distinct()` here. I think it's clearer to use `Distinct()` unless you need to keep a set around longer term.